packages/assets-controllers/src/MultichainTokensRatesController/types.ts

```diff
@@ -0,0 +1,38 @@
+import type { CaipAssetId } from '@metamask/utils';
+
+/**
+ * Represents the rate details for a given token conversion.
+ */
+export type RateData = {
+  conversionTime: number;
+  rate: string;
+};
+
+/**
+ * Represents the conversion rates for a specific token.
+ * The key is a currency identifier (for example, 'swift:0/iso4217:USD'),
+ * and the value is a RateData object.
+ */
+export type TokenConversionRates = {
+  [currency: CaipAssetId]: RateData;
```

Comment:
Could we use `AssetConversion` from `@metamask/snaps-sdk` rather than re-defining a type ? https://github.com/MetaMask/snaps/blob/361cd5b5b877520d9a2a5bd5c109dc31917c8b14/packages/snaps-sdk/src/types/handlers/assets-conversion.ts#L25